.gitignore

```diff
@@ -68,3 +68,9 @@ GTAGS
 # dashboard
 /src/pybind/mgr/dashboard/frontend/src/environments/environment.ts
 /src/pybind/mgr/dashboard/frontend/src/environments/environment.prod.ts
+.classpath
```

Comment:
@adamyanova why are these being added to .gitignore?

Reply:
@alimaredia  Which one do you refer to?
The *ts files come from the master branch. I myself have not explicitly added them. 
The .classpath is created from the editor I used, hence I ignored it.

Reply:
I was talking about the `.classpath`, `.project`, and the ones below that. Can the changes to this file be removed?

Reply:
done